src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/token/Nonce.java

```diff
@@ -12,11 +12,15 @@
 @Getter
 @RequiredArgsConstructor
 class Nonce {
+
+    @Getter
     @NonNull
-    final String value;
-    final String count = "1"; //token should use Nonce only once
+    private final String value;
+
+    @Getter
+    private static final String COUNT = "1"; //token should use Nonce only once
```

Comment:
We wouldn't normally use a getter for a static final member.
```suggestion
    public static final String COUNT = "1"; //token should use Nonce only once
```
The `getCOUNT()` generated name could be a hint that something's off!

Reply:
Implemented in a6dbd36